src/services/projects.service.js

```diff
@@ -310,6 +311,14 @@ export default class ProjectService {
     this.log.info(`ProjectService: delete contributor ${user} from project with id ${projectId}`);
     const project = await this.getProjectById(projectId);
 
+    // convert the result from the sequelize query to a string, the parse the json.
+    const owners = JSON.parse(JSON.stringify(await project.getOwners()));
```

Comment:
Why do we stringify it then parse it?

Reply:
Sequelize returns it in a format that isn't able to check the length. So Stringifying it turns it into a format that has an array and then parasing it allows us to check the length.

Reply:
Basically calling `.length` or `.count` doesn't work on `project.getOwners()`